source/gems/dry-component/booting-a-dependency.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Booting a Dependency
 layout: gem-single
+name: dry-component
```

Comment:
Assuming `page.parent` works correctly for index pages, we should be able to specify `name` of a page only in the index pages and do `current_page.data.name || current.page.parent.data.name` to reduce duplication.